package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
```

Comment:
This diff means we make a mistake on the last release: the package.json and package-lock.json must be in sync. I thought `npm ci` would alert us of this, but unfortunately that is not the case. Next time If we use `npm version` instead of manually editing version definition, it's going to be ok.

To keep a good repo history, this fix should be on a specific commit. I just created https://github.com/aziontech/azion-framework-adapter/pull/50 for this purpose.